src/content/reference/okteto-cli.mdx

```diff
@@ -142,14 +142,12 @@ Use the arrow keys to navigate: ↓ ↑ → ←
 When you run `okteto context`, an account will be created for your own `URL` if it's the first time you set the context in.
 The CLI will exchange an authorization token with `URL` and save your API token and Okteto certificates information under `$HOME/.okteto/`.
 
-#### Using Environment Variables to authenticate
 
-Environment variables provide another way to specify your credentials. This is the recommended method when scripting tasks or when [creating preview environments](previews/index.mdx).
+:::tip
 
-The supported environment variables are:
+In your CI/CD you should use environment variables to specify your Okteto Context, Namespace and token. Check out [Environment Variables](/docs/1.20/core/credentials/environment-variables/) to know more.
```

Comment:
Links shouldn't be to a specific version. Something like `(core/credentials/environment-variables.mdx)` should work